packages/components/style.css

```diff
@@ -1,23 +1,26 @@
 @layer l-base, l-nextra, l-components;
 
-@import 'tailwindcss/base' layer(l-base);
+/*@import 'tailwindcss/base' layer(l-base);*/
 @import 'nextra-theme-docs/dist/style.css' layer(l-nextra);
 @import 'tailwindcss/components' layer(l-components);
 @import 'tailwindcss/utilities';
 
 .nextra-banner-container._bg-neutral-900._flex {
   @apply bg-[#48224e] bg-none;
 }
+
 :root {
   --nextra-navbar-height: 72px;
 }
 
-.nextra-nav-container [role='menu']:has(a[href='https://the-guild.dev/graphql/hive'])
-{
-  @apply grid max-h-[unset] w-[470px] grid-cols-3 gap-y-2 p-2;
+@layer l-nextra {
+  #headlessui-portal-root [role='menu']:has(a[href='https://the-guild.dev/graphql/hive'])
```

Comment:
Won't this break if any of the websites installs Headless UI independently and renders a menu with a link to hive?

Reply:
This code is only for overriding Nextra's `menu`, I think we'll throw it away soon when we use the shared Hive header

<img width="1040" alt="image" src="https://github.com/user-attachments/assets/e3ef5145-4413-4a88-a4a4-39c8395c72e9">